### DISCUSSION TITLE
Problem diabat.mpq

### DISCUSSION THREAD

**Discussion by isidoro-80** [diasurgical/DevilutionX#2483] [Q&A] [answered]
Hello i install devilutionx in rpi4. I start program with the command from shell
devilutionx --data-dir ... (directory to file diabdat.mpq)
but the program not work. I read the message unable to open main data archive diabdat.mpq.
Help me please thanks

**Comment by AJenbo** [comment]
Since you left out the actual paths it's a bit hard to see what is going on.
--data-dir should point to the folder not the file.

**Reply by isidoro-80** [reply]
this is the command devilutionx --data-dir /home/pi/RetroPie/roms/ports/devilutionx/ 
this is the directory of diabdat.mpq  /home/pi/RetroPie/roms/ports/devilutionx/ 
I do not understand why it does not see the file I think it is a program error. the diabat file is correct the game in windows works.

**Reply by AJenbo** [reply]
Did you rename it as `diabat.mpq`, if that is the case then that is the problem.

If the mpq is in the same folder as the application you do not need to provide the path.

I don't think this is an application error or more people would be encountering it, it's most likely an issue with you setup.

Please use `ls` or similar to show the actual full paths.

**Comment by isidoro-80** [comment]
il file diabdat.mpq is ok forgive for the spelling mistake it is written in lowercase.
I did two tests
first test 
to test I created a folder with all the files contained in devilutionx-linux-armhf.tar.xz ver 1.2.1 in /home/pi/.local/share/diasurgical/devilution/ 
the file are: CharisSILB.ttf, devilutionx, devilutionx.mpq, **diabdat.mpq**
then I copied the files contained in data.tar into their respective folders usr/bin and usr/share
then i write ./devilutionx and I read the message unable to open main data archive diabdat.mpq.

second test
i installed devilutionx from retropie setup but the installation stops. the files have been copied for only a part. I proceeded to insert the files devilutionx-linux-armhf.tar.xz ver 1.2.1 in their respective folders.
i have this file:
devilutionx - Diablo Engine.sh in home/pi/Retropie/roms/ports/
**diabdat.mpq** in home/pi/RetroPie/roms/ports/devilutionx
devilution.cfg in opt/retropie/configs/ports/devilutionx
CharisSILB.ttf, devilutionx, devilutionx.mpq in opt/retropie/ports/devilutionx/ and in opt/retropie/configs/ports/devilutionx/devilution
then from emulationstation I launched the devilutionx - diablo engine.sh file from port gamelist and I read the message unable to open main data archive diabdat.mpq.

**Reply by glebm** [reply]
`engine.sh` and `devilution.cfg` are not DevilutionX's files. Are they from your distro? Perhaps the problem is in them?

**Comment by isidoro-80** [comment, marked as answer]
hello sorry now everything ok. I solved my problem. the error was in the diabdat.mpq file. not so why but it was perfectly taken from a working cd. taken from another cd now everything works
